package.json

```diff
@@ -19,6 +19,9 @@
     "automerge-repo-react-hooks": "^0.0.54",
     "automerge-repo-storage-localforage": "^0.0.56",
     "eventemitter3": "^5.0.1",
+    "i": "^0.3.7",
+    "npm": "^9.7.2",
```

Comment:
I don't think "i" and "npm" are supposed to be here but we can can clean them up later with "depcheck", cause there might be other dependencies we're not using.

Reply:
yeah ... let me remove those ones